File: Python/debug/pause_script.py
from playwright.sync_api import sync_playwright

def main():
    with sync_playwright() as p:
        browser = p.chromium.launch()
        page = browser.new_page()
        page.goto("https://example.com")
        page.pause()  # Opens inspector in terminal
        browser.close()

main()

File: NodeJS/debug/pause_script.js
const { chromium } = require('playwright');

(async () => {
  const browser = await chromium.launch({ headless: false });
  const page = await browser.newPage();
  await page.goto('https://example.com');
  await page.pause(); // Triggers debugger
  await browser.close();
})();
